scripts/content.js

```diff
@@ -353,6 +360,18 @@ const DEFAULT_ELEMENTS = [
   },
 ];
 
+let removeElements = false;
+
+chrome.runtime.onMessage.addListener((message) => {
+  if (message.action === "set-hide-elements") {
+    removeElements = false;
+    console.log("Hide elements mode activated.");
```

Comment:
you should remove all the console.log() if your PR is ready, no need to pollute the console